schema/mysql/upgrades/dependencies.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TABLE `dependency` (
+  `id` binary(20) NOT NULL,
+  `name` text NOT NULL,
+  `display_name` text NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin ROW_FORMAT=DYNAMIC;
+
+CREATE TABLE `dependency_state` (
+  `id` binary(20) NOT NULL,
+  `dependency_id` binary(20) NOT NULL,
+  `failed` enum('n', 'y') NOT NULL,
+  UNIQUE INDEX `dependency_state_dependency_id_uindex` (dependency_id),
+  KEY `dependency_state_dependency_id_fk` (`dependency_id`),
+  CONSTRAINT `dependency_state_dependency_id_fk` FOREIGN KEY (`dependency_id`) REFERENCES `dependency` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin ROW_FORMAT=DYNAMIC;
+
+CREATE TABLE `redundancy_group` (
+  `id` binary(20) NOT NULL,
+  `name` text NOT NULL,
+  `display_name` text NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin ROW_FORMAT=DYNAMIC;
+
+CREATE TABLE `redundancy_group_state` (
+  `id` binary(20) NOT NULL,
+  `redundancy_group_id` binary(20) NOT NULL,
+  `failed` enum('n', 'y') NOT NULL,
```

Comment:
The `redundancy_group_state` table is missing `last_state_change` column, to track how long since the redundancy group has failed.

Reply:
done

Reply:
It seems that now it's there twice. Please refer to my other comment.